### PR TITLE
[selenium-webdriver] ran dprint fmt

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
👋 Hi! This PR is meant to let you know that we're going to be applying formatting automatically to all DefinitelyTyped packages[^1]. This PR is split out of:

* https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/65993: the discussion that includes backing context and our planned approach
* https://github.com/DefinitelyTyped/DefinitelyTyped/pull/66235: onboards the repository to [dprint](https://dprint.dev)

This package is particularly huge (>1MB), so we're tagging you in an individual PR as a heads up. Please post here to let us know if you have any workflow needs that auto-formatting might disrupt[^2].

Cheers! 💖 

[^1]: Note that formatting will be applied as a commit to `master` after each PR. PRs won't be blocked on any CI check for proper formatting. See #65993 for more information.
[^2]: For example, if this package is fully auto-generated (rather than manually edited), subsequent PRs might have a lot of noise from formatting differences. We can exclude it from formatting in that case.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## selenium-webdriver ([<kbd>unpkg</kbd>](https://unpkg.com/browse/selenium-webdriver@latest/))
was missing the following properties:
1. ThenableWebDriver
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 4ed948f776725f3109a64369af99b2a5a78cf308
</p>

**Review by SupernaviX — APPROVED** [review]

**Review by shubhsheth — APPROVED** [review]

File: types/selenium-webdriver/devtools/networkinterceptor.d.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ declare class HttpResponse {
      * @param {*} value
      */
     addHeaders(header: string, value: any): void;
-    get headers(): Array<{name: string, value: any}>;
+    get headers(): Array<{ name: string; value: any }>;
```
Comment (AutomatedTester) [inline types/selenium-webdriver/devtools/networkinterceptor.d.ts:20]: Are semi-colons allowed between args?

Comment (JoshuaKGoldberg) [reply types/selenium-webdriver/devtools/networkinterceptor.d.ts:20]: Between args, no. But this is inside an object literal (`{ ... }`).

Comment (AutomatedTester) [reply types/selenium-webdriver/devtools/networkinterceptor.d.ts:20]: 🤦 I misread the code. Thanks for explaining

**Review by AutomatedTester — APPROVED** [review]

**Review by andrewbranch — APPROVED** [review]

**Comment by JoshuaKGoldberg** [comment]
Ready to merge

**Pull request merged**